src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -830,10 +830,10 @@ type TaskSeq =
     static member takeWhile: predicate: ('T -> bool) -> source: TaskSeq<'T> -> TaskSeq<'T>
 
     /// <summary>
-    /// Returns a sequence that, when iterated, yields elements of the underlying sequence while the
+    /// Returns a task sequence that, when iterated, yields elements of the underlying sequence while the
     /// given asynchronous function <paramref name="predicate" /> returns <see cref="true" />, and then returns no further elements.
     /// The first element where the predicate returns <see cref="false" /> is not included in the resulting sequence
```

Comment:
while this might work well for flicking between two functions in intellisense, something terser might work for whats afterall the normal case:
```suggestion
    /// Stops consuming the source and yielding items as soon as the predicate returns <c>false</c>.
```

Reply:
I struggled with this one, thanks, I like it much more!

Reply:
Fixed for both functions